sdk/lib/delayimp/delayimp.c

```diff
@@ -16,8 +14,8 @@
 /**** Linker magic: provide a default (NULL) pointer, but allow the user to override it ****/
 
 /* The actual items we use */
-PfnDliHook __pfnDliNotifyHook2;
-PfnDliHook __pfnDliFailureHook2;
+extern PfnDliHook __pfnDliNotifyHook2;
+extern PfnDliHook __pfnDliFailureHook2;
```

Comment:
@tkreuzer I was fixing this thing here: https://github.com/reactos/reactos/pull/7784

Reply:
If it works, that is a reasonable alternative. So are you going to finish it?

Reply:
yes but to be fair, that PR will probably be merged in 3 years.

Reply:
It needs the extra tests asked by Mark. I'll try to speed this up since I also need it for other purposes.